ietf/meeting/helpers.py

```diff
@@ -124,7 +124,7 @@ def preprocess_assignments_for_agenda(assignments_queryset, meeting, extra_prefe
             # check before blindly assigning to meeting just in case.
             if a.session.meeting.pk == meeting.pk:
                 a.session.meeting = meeting
-            a.session.order_number = None
+            a.session.order_number = a.session.order_in_meeting() if a.session.group else None
```

Comment:
I know this was preserving existing conditionals, but I wonder if it's possible for session group to be falsy with the current code?

Reply:
It shouldn't be possible - the relation from `Session` to `Group` is `group = ForeignKey(Group)`, which does not allow nulls.